src/main/kotlin/com/nextstep/jngcii/racingcar/view/ResultView.kt

```diff
@@ -5,7 +5,19 @@ class ResultView {
         println("실행 결과")
     }
 
-    fun printRoad(distanceShape: String) {
-        println(distanceShape)
+    fun printDistance(name: String, distance: Int) {
+        println("$name : ${distance.shape}")
```

Comment:
```suggestion
        println("$name : ${DASH.repeat(distance)}")
```
repeat 확장 함수를 이용하면 간단하게 구현할 수 있습니다!

Reply:
[refactor : DASH repeat 로직 리팩터링](https://github.com/next-step/kotlin-racingcar/pull/625/commits/d7557eddabef0bd3ed0b85117dbbcad00ad730e9)

수정했습니다!